src/Microsoft.DotNet.Interactive.Tests/LanguageKernelPackageTests.cs

```diff
@@ -457,11 +450,8 @@ public async Task Pound_i_nuget_with_multi_submissions_combines_the_Text_Produce
 
             using var events = result.KernelEvents.ToSubscribedList();
 
-            // For the DisplayedValueUpdated events strip out the restore sources
-            // Verify that they match the expected values
-            events.Should()
-                  .ContainSingle<DisplayedValueProduced>()
-                  .Which.FormattedValues
+            events.OfType<DisplayedValueProduced>()
+                  .First().FormattedValues
```

Comment:
Assuming we're expecting exactly one `DisplayedValueProduced` event, this is better written like this to provide better output:

```cs
events.Should().ContainSingle<DisplayedValueProduced>(e => e.MimeType)
    .Which.Value
    .Should()
    .ContainAll(expectedList);
```

Reply:
I think there are two displayValueProduced.
1 for restore sources and 1 for packages loaded.

Reply:
now we produce a single displayValueProduced for aeach restore